hack/lib/protoc.sh

```diff
@@ -76,13 +93,32 @@ function kube::protoc::protoc() {
   )
 }
 
+# Generates $1/api.pb.go from the protobuf file $1/api.proto without using gogo
+# $1: Full path to the directory where the api.proto file is
+function kube::protoc::protoc() {
+  local package=${1}
+
+  (
+    cd "${package}"
+
+    protoc \
```

Comment:
I'm surprised we don't need the equivalent of 
```
      --proto_path="${KUBE_ROOT}/vendor" \
      --proto_path="${KUBE_ROOT}/staging/src" \
      --proto_path="${KUBE_ROOT}/third_party/protobuf" \
```

Reply:
At least not for the CRI, where we do not import anything from there.

Reply:
I'm +100 for moving generic protobindings away from gogo, but we should make these generator functions suitable for all the generic protobindings to call

Reply:
That would impact generated go code but not the wire serialization... getting rid of the gogo dependency would be worth the go churn, I think.

Reply:
A possible smaller first step would be to drop the gogo customizations and the gogo proto requirement from the api.proto files

Reply:
I think we can iterate here. Starting with the CRI and following up with the other parts to ensure we don't have too many moving parts.